module/mailnews/mailfile.c

```diff
@@ -126,12 +128,15 @@ void MailNewMessage(int server_index, char *sender, int num_recipients,
 {
    int index, num_msgs, msgnum;
    char new_msg[MAXMAIL + 200 + MAX_SUBJECT + MAXUSERNAME * MAX_RECIPIENTS];
-   char *subject, *ptr, *subject_str;
+   int subject_ids[] = {IDS_SUBJECT_ENGLISH, IDS_SUBJECT_GERMAN, IDS_SUBJECT_PORTUGUESE};
```

Comment:
Make const

Reply:
The likelihood of 3 more language anytime soon seems low, but would you like me to add support for more than 5 strings here?

Reply:
Yeah, we should avoid relying on those terrible static bufferes.

Reply:
I gave it a go.